app/guests/[id].tsx

```diff
@@ -44,7 +44,7 @@ export default function Guests() {
         if (typeof id !== 'string') {
           return;
         }
-        const response = await fetch(`/${id}`);
+        const response = await fetch(`/api/${id}`);
```

Comment:
If we agree on `guestId`

Reply:
Agreed on `guestId` 
[aaf6515](https://github.com/upleveled/expo-example-spring-2024-atvie/pull/6/commits/aaf6515b128bdf9be2723503de899a85166f993f)